### DISCUSSION TITLE
Light blocks

### DISCUSSION THREAD

**Discussion by StrideyGH** [Roadhog360/Et-Futurum-Requiem#284] [Ideas]
The invisible light blocks from 1.17 would be REALLY useful :D
https://minecraft.fandom.com/wiki/Light_Block

(speaking of invisible blocks, barriers would be great too :P)

**Comment by GregoriusT** [comment]
Wait uh what exactly do you mean by Light Blocks? Can you list them? Do you mean any type of Light emitting Blocks?

**Reply by StrideyGH** [reply]
Sorry, I should've been clearer. I meant the invisible light level blocks https://minecraft.fandom.com/wiki/Light_Block
I've edited the post to be clearer

**Comment by DarianLStephens** [comment]
The only light blocks added in 1.17 were the Glow Berry vines, Candles, Glow Lichen...
Oh, I know what they mean! The actual 'Light' blocks that are creative-only, providing light without a visible block.

**Reply by StrideyGH** [reply]
yeah, the invisible ones. I've edited the post now, I should've made it clearer :P

**Comment by Roadhog360** [comment]
This request has been fulfilled as of [14cf825](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/14cf825363baf15800a3497d3e006db2c9dd5ff0)
